classes/Form.php

```diff
@@ -841,6 +841,9 @@ public function post()
                 if ($field['type'] === 'checkbox' || $field['type'] === 'switch') {
                     $data[$name] = isset($data[$name]) ? true : false;
                 }
+                if ($field['type'] === 'email' && isset($data[$name])) {
```

Comment:
I disagree with this. Emails should be written as they were. The logic that handles the rest should be case insensitive instead.

This is same as usernames -- in flex I allow usernames to contain mixed case, they are just normalized in the index.

Also this code is in wrong place -- post() will not be called for every form.